media/redesign/stylus/components/wiki/revision-list.styl

```diff
@@ -97,3 +97,7 @@ html[dir=rtl] .revision-list-controls:before {
     width: 460px;
 }
 
+/* make the Compare selected revisions button align to layout */
+.revision-list-controls .fixed {
+    left: 24px;
```

Comment:
I think I've added this correctly in my recent commit. It's a bit hard to test though. Mind taking another look now?